package.json

```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@std/esm": "^0.14.0",
-    "funcadelic": "^0.2.6",
+    "funcadelic": "0.2.6",
```

Comment:
Curious why.

Reply:
It's sliding to 0.2.7. I'll need to investigate why 0.2.7 is breaking tests.